app/src/main/java/hm/orz/chaos114/android/tumekyouen/modules/title/TitleActivity.java

```diff
@@ -83,9 +87,23 @@ public void onCreate(final Bundle savedInstanceState) {
         binding.adView.loadAd(adRequest);
 
         final TwitterAuthToken loginInfo = loginUtil.loadLoginInfo();
+        Timber.d("loginInfo = %s", loginInfo);
         if (loginInfo != null) {
             // 認証情報が存在する場合
-            new ServerRegistTask().execute(loginInfo);
+            binding.connectButton.setEnabled(false);
+            tumeKyouenService.login(loginInfo.token, loginInfo.secret)
+                    .subscribeOn(Schedulers.io())
+                    .observeOn(AndroidSchedulers.mainThread())
+                    .subscribe(s -> {
+                                Timber.d("sucess : %s", s);
+                                binding.connectButton.setEnabled(true);
+                                // 成功した場合
+                                onSuccessTwitterAuth();
+                            },
+                            throwable -> {
+                                Timber.d("fail", throwable);
```

Comment:
ERROR: [TimberArgCount] Wrong argument count, format string `fail` requires `0` but format call supplies `1`
 When a formatted string takes arguments, you need to pass at least that amount of arguments to the formatting call.